form-editor-cae/src/main/java/com/tallence/formeditor/cae/validator/TextValidator.java

```diff
@@ -87,7 +87,7 @@ public Pattern getRegexp() {
   }
 
   public void setRegexp(String regexp) {
-    if (regexp != null) {
+    if (StringUtils.hasText(regexp)) {
```

Comment:
We should probably change this to handle `PatternSyntaxException` too.

Reply:
Wie besprochen. Sinnvoll wäre wohl ein Studio-Validator bei der Eingabe des regexp-Strings im Studio durch den Redakteur. Siehe:
#26